server/src/inmantals/server.py

```diff
@@ -118,22 +125,83 @@ def flatten(self, line, char):
         assert char < 100000
         return line * 100000 + char
 
+    def create_tmp_project(self) -> str:
+        ten_random_digits = "".join(random.choice(string.digits) for _ in range(10))
+        project_name = "project" + ten_random_digits
```

Comment:
I'm not 100% sure about this random name thing. One the one hand it allows opening 2 different modules in 2 different windows. But on the other hand I think my cleaning logic is not good enough at the moment and we are leaking some projects that are never cleaned up in case the `shutdown` method isn't called.